File: 백준/Bronze/35097. 2025/2025.py
import sys
input=lambda:sys.stdin.readline().rstrip("\n")
while True:
    n=int(input())
    if n==0: break
    rlt=0
    for i in range(1,n+1):
        for j in range(1,n+1):
            rlt+=i*j
    print(rlt)

File: 백준/Bronze/35097. 2025/2025.cc
#include <iostream>
using namespace std;
int main(void) {
    int n, rlt;
    while (true) {
        cin >> n;
        if (n == 0) break;
        rlt = 0;
        for (int i = 1; i < n + 1; i++) {
            for (int j = 1; j < n + 1; j++) {
                rlt += i * j;
            }
        }
        cout << rlt << " \n";
    }
    return 0;
}